tests/api/test_mod_corpus_association.py

```diff
@@ -0,0 +1,84 @@
+import pytest
+from starlette.testclient import TestClient
+from fastapi import status
+
+from agr_literature_service.api.main import app
+from .fixtures import auth_headers, db # noqa
+from .test_reference import create_test_reference # noqa
+from .test_mod import create_test_mod # noqa
+from collections import namedtuple
+
+create_test_reference2 = create_test_reference
+
+TestMCAData = namedtuple('TestMCAData', ['response', 'new_mca_id', 'related_ref_curie'])
+
+
+@pytest.fixture
+def create_test_mca(db, auth_headers, create_test_reference, create_test_mod): # noqa
+    print("***** Adding a test mod-corpus association *****")
+    with TestClient(app) as client:
+        mod_response = client.get(url=f"/mod/{create_test_mod.new_mod_abbreviation}")
+        mod_abbreviation = mod_response.json()["abbreviation"]
+        new_mcc = {
```

Comment:
new_mca would probably be clearer, but it makes sense

Reply:
I'll change it to mca, probably a typo